### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by smiller171**
I've added a dockerfile for running this on systems without Go installed. Unfortunately it doesn't convert correctly for me.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM golang as test
```
Comment (tmccombs) [inline Dockerfile]: What is the purpose of copying everything into the test layer, just to copy it over to the build layer?

Comment (sirlatrom) [reply Dockerfile:1]: That's a good question. If I create a new PR with just the build and final stages (i.e. test and build combined into one), would you be willing to set up Docker Hub auto builds?

Comment (tmccombs) [reply Dockerfile:1]: sure

**Comment by tmccombs** [comment]
This was done in #9

**Pull request closed without merging**